data/types/food/food_test.go

```diff
@@ -559,6 +564,11 @@ var _ = Describe("Food", func() {
 						datum.PrescribedNutrition = NewNutrition()
 					},
 				),
+				Entry("does not crash when prescriptor is not set",
+					func(datum *food.Food) {
+						datum.Prescriptor = common.NewPrescriptor()
```

Comment:
```suggestion
						datum.Prescriptor.Prescriptor = nil
```
To make the old code crash